tools/print_all_namespace.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
This script can be combined with https://github.com/eclipse-iceoryx/iceoryx/blob/master/tools/list_stl_dependencies.sh

Reply:
@dkroenke Let's keep the scripts separate for now and move them into a directory in your follow-up PR.